### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by praveenreddypingili1381996**

### CODE REVIEW

**Comment by madhulika9293** [comment]
Please do task3 ie., import books.csv to the database. Do that and I'll review again.

**Comment by praveenreddypingili1381996** [comment]
I completed task 3 and pushed to github.Please find it.

File: project1/templates/import.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    f = open("books.csv")
+    reader = csv.reader(f)
+    counter = 0
+    for isbn, title, author, pub_year in reader:
+        if isbn == "isbn":
+            continue
+        counter += 1
+        book = Book(id=counter, isbn=isbn, title=title, author=author, pub_year=int(pub_year))
+        db.session.add(book)
+        print(f"{counter}) Added {title} with {isbn} written by {author} in {pub_year}.")
```
Comment (madhulika9293) [inline project1/templates/import.py:30]: You don't have to print. For now, its okay. Don't do this in future codes!

**Review by madhulika9293 — COMMENTED** [review]
Good job! Your Day 10 tasks are over. You can merge to master now.

**Pull request merged**